utils/cLog.cpp

```diff
@@ -165,12 +165,12 @@ void cLogger::log(eLogLevel level, eLogComponent component, const std::string& e
 
     logline += event;
 
-    fprintf(m_file, "%s\n", logline.c_str()); // print the text into the file
-    fflush(m_file);
+    logCommentLine(logline);
```

Comment:
The comment line function is to log text with two backslaches in front of it.
TBH, I'm not a big fan of it, but Stefan likes it to have "headers" before certain blocks of log lines.
What do you say, @stefanhendriks, remove this feature? We log the component with each log line, that may be enough.
I'm OK either way.

Reply:
I fix as previously